financial-services-accelerator/accelerators/fs-is/repository/resources/apis/accelerator-extensions-v1.yaml

```diff
@@ -1602,13 +1602,23 @@ components:
       properties:
         consentId:
           type: string
+        type:
```

Comment:
Why the type needed? Isn't it better to add type Nd consent id inside consent resources

Reply:
Consent id is added as an identifier to the DTO. Type is needed here because consent resource will be null if the consent is not pre initiated.